### DISCUSSION TITLE
Auto-skip doesn't skip the intro multiple times when it's required

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#21083] [Q&A] [answered]
As title. Here's an example [with the map "MAYDAY"](https://osu.ppy.sh/beatmapsets/754219#osu/1587909) (which has more than one skip in the intro). It auto-skips only once.
The feature should probably trigger all the available initial skips, I think.

https://user-images.githubusercontent.com/43073074/199621720-118aee39-e12b-48d4-8849-ea2ab04e9291.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/21087
